packages/grafana-llm-app/src/plugin.json

```diff
@@ -42,6 +42,30 @@
     "permissions": [
       {
         "action": "orgs:read"
+      },
+      {
+        "action": "datasources:read",
+        "scope": "datasources:uid:*"
```

Comment:
Do we need the `:uid` here or just `datasources:*`? I see the latter on https://grafana.com/docs/grafana/latest/developers/http_api/data_source/.

Reply:
Good question, I figured both would work but have updated to match the docs.